pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -837,3 +842,26 @@ func TestHighProirotyBackoff(t *testing.T) {
 		t.Errorf("Expected to get mid prority pod, got: %v", p)
 	}
 }
+
+// TestHighProirotyFlushUnschedulableQLeftover tests that pods will be moved to
+// activeQ after one minutes if it is in unschedulableQ
+func TestHighProirotyFlushUnschedulableQLeftover(t *testing.T) {
+	q := NewPriorityQueue(nil)
+	q.unschedulableQ.addOrUpdate(&highPriorityPod)
+	q.unschedulableQ.addOrUpdate(&medPriorityPod)
+	q.unschedulableQ.addOrUpdate(&unschedulablePod)
+
+	highPriorityPod.CreationTimestamp = metav1.Time{Time: time.Now().Add(-1 * unschedulableQTimeInterval)}
```

Comment:
Please do not modify the global objects. Copy them in the test before modifying them.
Also, please use pod status LastProbeTime and LastTransitionTime in the test as well. See `TestPodFailedSchedulingMultipleTimesDoesNotBlockNewerPod` for examples.